### ISSUE TITLE
[STORM-950] Apache Storm website redesign

### ISSUE THREAD

**Issue by jira-importer** [apache/storm#4788]
No Description

---
<details><summary><i>Originally reported by <a title="Harsha" href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=sriharsha">sriharsha</a>, imported from: <a href="https://issues.apache.org/jira/browse/STORM-950" target="_blank">Apache Storm website redesign</a></i></summary>
<i><ul>
<li><b>assignee</b>: <a title="Harsha" href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=sriharsha">sriharsha</a>
<li><b>status</b>: Resolved
<li><b>priority</b>: Major
<li><b>resolution</b>: Fixed
<li><b>resolved</b>: 2015-09-11T19:33:03+01:00
<li><b>imported</b>: 2025-01-24
</ul></i>
</details>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=sriharsha">sriharsha</a>:</i>
<p>Here is staging website with updated design<br/>
<a href="http://people.apache.org/~sriharsha/apache-storm-ws/" class="external-link" target="_blank" rel="nofollow noopener">http://people.apache.org/~sriharsha/apache-storm-ws/</a></p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=dagit">dagit</a>:</i>
<p>This is beautiful, nice work!</p>

<p>If I scroll to the bottom of the page in FF, I see a lot of glitches, and I cannot see the full footer on the page.  Not sure if it is just me, do you see the same?</p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=sriharsha">sriharsha</a>:</i>
<p><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=dagit" class="user-hover" rel="dagit">Derek Dagit</a> I tried it on Firefox 40 and Chrome 43 it looks good to me. Can you give me your firefox version.</p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=dagit">dagit</a>:</i>
<p>Yeah, its FF 39.0 on OSX.</p>

<p>The size of the canvas is 1022x1325-- the issue might happen only for certain window sizes...or maybe it's only my browser.</p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=dagit">dagit</a>:</i>
<p>OK, just saw it happen for another user with a similar setup.</p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=dagit">dagit</a>:</i>
<p>When the header bar attaches to the top of the window, the whole page content jumps.  That is what is causing it I think, if the window height is in a certain range.</p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=ptgoetz">ptgoetz</a>:</i>
<p>Keep in mind this is just a preview at the moment. We haven't yet converted it over to Jekyll (to allow documentation to be authored in Markdown format).</p>

<p>Once that is done we'll open up a pull request.</p>

<p>For now we're just looking for feedback on the look and feel, browser compatibility, etc.</p>

<p>It's designed to be responsive as well, so we should test with tablets/mobile devices as well.</p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=sriharsha">sriharsha</a>:</i>
<p>Thanks <a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=dagit" class="user-hover" rel="dagit">Derek Dagit</a>. I'll try to reproduce this. If its possible to take a screenshot of the issue can you please update the jira.</p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=kabhwan">kabhwan</a>:</i>
<p>Looks beautiful on my laptop!<br/>
Personally I really love that it exposes "Manual" page's contents to "Documentation" page's tabs.</p>

<p>Looks beautiful on my iPhone, too.<br/>
How about adjusting the order of main page? I think "Why use Storm?" and "Latest News" are more important than "Tweets", but it's just 2 cents.</p>

<p>ps. Maybe off-topic, JavaDoc is exposed to the first line from "Basics of Storm" tab, but it links 0.8.x javadocs page. <br/>
The page should be removed, or we should create new page which includes each apache version's javadoc, and link to new page.</p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=sriharsha">sriharsha</a>:</i>
<p><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=kabhwan" class="user-hover" rel="kabhwan">Jungtaek Lim</a> Thanks for the feedback.  Yes we should be changing the order for the mobile devices<br/>
Regarding javadoc and other links issue is its still the old content. I'll try to get it fixed by our team here as much as possible. </p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=linyuxin">linyuxin</a>:</i>
<p>Looks beautiful and hierarchical. <br/>
But the font may be too "thin" to me. It looks a little obscure.</p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>GitHub user harshach opened a pull request:</p>

<p>    <a href="https://github.com/apache/storm/pull/654" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/654</a></p>

<p>    <a href="https://issues.apache.org/jira/browse/STORM-950" title="Apache Storm website redesign" class="issue-link" data-issue-key="STORM-950"><del>STORM-950</del></a>. Apache Storm website redesign.</p>

<p>    To test this.  Install jekyll<br/>
    on OS X sudo gem install jekyll<br/>
    cd storm/docs/<br/>
    jekyll serve -w</p>

<p>You can merge this pull request into a Git repository by running:</p>

<p>    $ git pull <a href="https://github.com/harshach/incubator-storm" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/harshach/incubator-storm</a> <a href="https://issues.apache.org/jira/browse/STORM-950" title="Apache Storm website redesign" class="issue-link" data-issue-key="STORM-950"><del>STORM-950</del></a>-V1</p>

<p>Alternatively you can review and apply these changes as the patch at:</p>

<p>    <a href="https://github.com/apache/storm/pull/654.patch" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/654.patch</a></p>

<p>To close this pull request, make a commit to your master/trunk branch<br/>
with (at least) the following in the commit message:</p>

<p>    This closes #654</p>

<hr />
<p>commit a238ebad9627841286dd8f03f69430afec8170a1<br/>
Author: Sriharsha Chintalapani <mail@harsha.io><br/>
Date:   2015-07-27T03:18:33Z</p>

<p>    <a href="https://issues.apache.org/jira/browse/STORM-950" title="Apache Storm website redesign" class="issue-link" data-issue-key="STORM-950"><del>STORM-950</del></a>. Apache Storm website redesign.</p>

<hr />

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=sriharsha">sriharsha</a>:</i>
<p>Thanks for the feedback. Our team converted the layout into jekyll templates. All of the feedback above is addressed.</p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>Github user HeartSaVioR commented on the pull request:</p>

<p>    <a href="https://github.com/apache/storm/pull/654#issuecomment-125104496" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/654#issuecomment-125104496</a></p>

<p>    I merged into my master and followed your description to run webserver.<br/>
    Some broken things are seen, here's a list.</p>

<p>    index.html</p>
<hr />
<p>    Latest News : All links go to news.html, which could be linked as proper news page.</p>

<p>    Downloads</p>
<hr />
<p>    Current Beta Release<br/>
    0.10.0-beta -> 0.10.0-beta1</p>

<p>    Better to remove indentation (whitespaces) from Storm dependency</p>

<p>    Rationale</p>
<hr />
<p>    It shows two different footers. Seems like first one needs to be removed. </p>

<p>    Creating a New Storm Project</p>
<hr />
<p>    It shows two different footers. Seems like first one needs to be removed. </p>

<p>    Concepts</p>
<hr />
<p>    favicon cannot be loaded. (404), It prints '<' to the top line.</p>

<p>    Not found (404) -> <a href="http://localhost:4000/documentation/Running-topologies-on-a-production-cluster.html" class="external-link" target="_blank" rel="nofollow noopener">http://localhost:4000/documentation/Running-topologies-on-a-production-cluster.html</a></p>

<p>    Getting help</p>
<hr />
<p>    Not found (404) -> <br/>
    <a href="http://localhost:4000/DSLs-and-multilang-adapters.html" class="external-link" target="_blank" rel="nofollow noopener">http://localhost:4000/DSLs-and-multilang-adapters.html</a><br/>
    <a href="http://localhost:4000/Contributing-to-Storm.html" class="external-link" target="_blank" rel="nofollow noopener">http://localhost:4000/Contributing-to-Storm.html</a><br/>
    <a href="http://localhost:4000/Project-ideas.html" class="external-link" target="_blank" rel="nofollow noopener">http://localhost:4000/Project-ideas.html</a><br/>
    <a href="http://localhost:4000/Spout-implementations.html" class="external-link" target="_blank" rel="nofollow noopener">http://localhost:4000/Spout-implementations.html</a><br/>
    <a href="http://localhost:4000/powered-by.html" class="external-link" target="_blank" rel="nofollow noopener">http://localhost:4000/powered-by.html</a></p>

<p>    Overall</p>
<hr />
<p>    Image resources seems not loaded so it is replaced by box icon character.<br/>
    (Maybe fontawesome is missing?)</p>

<p>    Many links are represented as ```javascript:void(0);``` and doesn’t work.<br/>
    checked with setting-up.html, contributing.html, tutorial.html, etc.</p>

<p>    Minor:</p>
<ul class="alternate" type="square">
	<li># anchor doesn’t work as permanent link (tested with bylaws page)</li>
</ul>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>GitHub user harshach opened a pull request:</p>

<p>    <a href="https://github.com/apache/storm/pull/659" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/659</a></p>

<p>    <a href="https://issues.apache.org/jira/browse/STORM-950" title="Apache Storm website redesign" class="issue-link" data-issue-key="STORM-950"><del>STORM-950</del></a>. Apache Storm website redesign.</p>



<p>You can merge this pull request into a Git repository by running:</p>

<p>    $ git pull <a href="https://github.com/harshach/incubator-storm" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/harshach/incubator-storm</a> <a href="https://issues.apache.org/jira/browse/STORM-950" title="Apache Storm website redesign" class="issue-link" data-issue-key="STORM-950"><del>STORM-950</del></a>-V4</p>

<p>Alternatively you can review and apply these changes as the patch at:</p>

<p>    <a href="https://github.com/apache/storm/pull/659.patch" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/659.patch</a></p>

<p>To close this pull request, make a commit to your master/trunk branch<br/>
with (at least) the following in the commit message:</p>

<p>    This closes #659</p>

<hr />
<p>commit 2e02fdabe7accfb4eecd6d33d9b6788a6b63a464<br/>
Author: Sriharsha Chintalapani <harsha@hortonworks.com><br/>
Date:   2015-07-30T18:20:52Z</p>

<p>    <a href="https://issues.apache.org/jira/browse/STORM-950" title="Apache Storm website redesign" class="issue-link" data-issue-key="STORM-950"><del>STORM-950</del></a>. Apache Storm website redesign.</p>

<hr />

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>Github user harshach closed the pull request at:</p>

<p>    <a href="https://github.com/apache/storm/pull/654" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/654</a></p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>Github user harshach commented on the pull request:</p>

<p>    <a href="https://github.com/apache/storm/pull/659#issuecomment-126424522" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/659#issuecomment-126424522</a></p>

<p>    @HeartSaVioR  your comments from previous one addressed in this one <a href="https://github.com/apache/storm/pull/654#issuecomment-125104496" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/654#issuecomment-125104496</a> . Please take a look</p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>Github user harshach commented on the pull request:</p>

<p>    <a href="https://github.com/apache/storm/pull/659#issuecomment-126424575" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/659#issuecomment-126424575</a></p>

<p>    @ptgoetz please take a look as well.</p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>Github user harshach closed the pull request at:</p>

<p>    <a href="https://github.com/apache/storm/pull/659" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/659</a></p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=kabhwan">kabhwan</a>:</i>
<p>Will you post new PR? Or closing newest PR is by mistake?</p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=sriharsha">sriharsha</a>:</i>
<p><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=kabhwan" class="user-hover" rel="kabhwan">Jungtaek Lim</a> will post a new PR. Previous PR still didn't address the html tags in markdown.</p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>GitHub user harshach opened a pull request:</p>

<p>    <a href="https://github.com/apache/storm/pull/663" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/663</a></p>

<p>    <a href="https://issues.apache.org/jira/browse/STORM-950" title="Apache Storm website redesign" class="issue-link" data-issue-key="STORM-950"><del>STORM-950</del></a>. Apache Storm website redesign.</p>

<p>    To run<br/>
    1. Install Jekyll<br/>
    2.  jekyll serve -w will start a webserver at localhost:4000</p>

<p>You can merge this pull request into a Git repository by running:</p>

<p>    $ git pull <a href="https://github.com/harshach/incubator-storm" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/harshach/incubator-storm</a> <a href="https://issues.apache.org/jira/browse/STORM-650" title="Storm-Kafka Refactoring and Improvements" class="issue-link" data-issue-key="STORM-650"><del>STORM-650</del></a>-V5</p>

<p>Alternatively you can review and apply these changes as the patch at:</p>

<p>    <a href="https://github.com/apache/storm/pull/663.patch" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/663.patch</a></p>

<p>To close this pull request, make a commit to your master/trunk branch<br/>
with (at least) the following in the commit message:</p>

<p>    This closes #663</p>

<hr />
<p>commit 0e88bda33c112b65c493fd9cdf16d3a9a2c2aab8<br/>
Author: Sriharsha Chintalapani <harsha@hortonworks.com><br/>
Date:   2015-07-31T19:43:49Z</p>

<p>    <a href="https://issues.apache.org/jira/browse/STORM-950" title="Apache Storm website redesign" class="issue-link" data-issue-key="STORM-950"><del>STORM-950</del></a>. Apache Storm website redesign.</p>

<hr />

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>Github user harshach commented on the pull request:</p>

<p>    <a href="https://github.com/apache/storm/pull/663#issuecomment-126795885" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/663#issuecomment-126795885</a></p>

<p>    @HeartSaVioR @ptgoetz please take a look.</p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>Github user HeartSaVioR commented on the pull request:</p>

<p>    <a href="https://github.com/apache/storm/pull/663#issuecomment-126848034" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/663#issuecomment-126848034</a></p>

<p>    @harshach <br/>
    Many resources and document links are broken. Maybe you forgot to add files to commit?</p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>Github user revans2 commented on the pull request:</p>

<p>    <a href="https://github.com/apache/storm/pull/663#issuecomment-128151202" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/663#issuecomment-128151202</a></p>

<p>    @harshach I am getting the same thing as @HeartSaVioR Just loading the main page has over 20 404 errors returned, many of them are images, but also .css and .js files.  Bootstrap also is complaining that jquery is not loaded.  One of those things that returned a 404.</p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>Github user harshach closed the pull request at:</p>

<p>    <a href="https://github.com/apache/storm/pull/663" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/663</a></p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>GitHub user harshach opened a pull request:</p>

<p>    <a href="https://github.com/apache/storm/pull/670" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/670</a></p>

<p>    <a href="https://issues.apache.org/jira/browse/STORM-950" title="Apache Storm website redesign" class="issue-link" data-issue-key="STORM-950"><del>STORM-950</del></a>. Apache Storm website redesign.</p>



<p>You can merge this pull request into a Git repository by running:</p>

<p>    $ git pull <a href="https://github.com/harshach/incubator-storm" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/harshach/incubator-storm</a> <a href="https://issues.apache.org/jira/browse/STORM-950" title="Apache Storm website redesign" class="issue-link" data-issue-key="STORM-950"><del>STORM-950</del></a>-V7</p>

<p>Alternatively you can review and apply these changes as the patch at:</p>

<p>    <a href="https://github.com/apache/storm/pull/670.patch" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/670.patch</a></p>

<p>To close this pull request, make a commit to your master/trunk branch<br/>
with (at least) the following in the commit message:</p>

<p>    This closes #670</p>

<hr />
<p>commit 4dfcfa04c949fc4514b304dd4dad1a5c1b8a8270<br/>
Author: Sriharsha Chintalapani <mail@harsha.io><br/>
Date:   2015-08-10T03:08:37Z</p>

<p>    <a href="https://issues.apache.org/jira/browse/STORM-950" title="Apache Storm website redesign" class="issue-link" data-issue-key="STORM-950"><del>STORM-950</del></a>. Apache Storm website redesign.</p>

<hr />

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>Github user harshach commented on the pull request:</p>

<p>    <a href="https://github.com/apache/storm/pull/670#issuecomment-129286293" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/670#issuecomment-129286293</a></p>

<p>    @revans2 how are you running this I didn't see any issues with resources or js missing 404s.<br/>
    @HeartSaVioR can you please check this PR .  I ran though the links didn't see any broken ones. Can you please point them out . Thanks.</p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>Github user harshach commented on the pull request:</p>

<p>    <a href="https://github.com/apache/storm/pull/670#issuecomment-129286465" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/670#issuecomment-129286465</a></p>

<p>    @HeartSaVioR @revans2 my bad looks like I missed a commit last time. Please try again. Thanks</p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>Github user HeartSaVioR commented on the pull request:</p>

<p>    <a href="https://github.com/apache/storm/pull/670#issuecomment-129340919" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/670#issuecomment-129340919</a></p>

<p>    @harshach It seems fixed. Thanks. I'll look into it.</p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>Github user HeartSaVioR commented on the pull request:</p>

<p>    <a href="https://github.com/apache/storm/pull/670#issuecomment-129460417" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/670#issuecomment-129460417</a></p>

<p>    @harshach <br/>
    Looks beautiful to me. And spots I reported seems be fixed.<br/>
    I looked at whole things and found new points to fix.</p>

<p>    downloads.html<br/>
    —</p>
<ul class="alternate" type="square">
	<li>Download link for “Current Beta Release" is still pointed to 0.10.0-beta, not 0.10.0-beta1.</li>
</ul>


<p>    multi-language.html<br/>
    —</p>
<ul class="alternate" type="square">
	<li>Adapters that implement this protocol exist for Ruby, Python, Javascript, Perl.</li>
	<li>Link of Ruby, Python, Javascript are broken. It should link to storm-multilang module.</li>
</ul>


<p>    documentation.html<br/>
    —</p>
<ul class="alternate" type="square">
	<li>"#" anchor doesn't work, so when we move to link and back, it always points to "basics of storm".</li>
</ul>


<p>    documentation.html#basic<br/>
    —</p>
<ul class="alternate" type="square">
	<li>Javadoc link points to Nathan’s github repo. It should be replaced to recent javadoc.</li>
</ul>


<p>    documentation/Serialization.html<br/>
    —</p>
<ul class="alternate" type="square">
	<li>We should replace all links about Kyro to github, <a href="https://github.com/EsotericSoftware/kryo" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/EsotericSoftware/kryo</a>.</li>
</ul>


<p>    documentation/Setting-up-a-Storm-cluster.html<br/>
    —</p>
<ul class="alternate" type="square">
	<li>The Storm releases can be downloaded from here.</li>
	<li>It would be better to link "Download" page instead of github releases.</li>
</ul>


<p>    documentation/Clojure-DSL.html<br/>
    —</p>
<ul class="alternate" type="square">
	<li>"backtype.storm.clojure" link points to 0.5.3 branch, which should point to master branch.</li>
	<li>"Direct groupings" links to its page, "Clojure-DSL", which doesn't explain direct groupings.</li>
</ul>


<p>    documentation/Metrics.html<br/>
    —</p>
<ul class="alternate" type="square">
	<li>"builtin metrics" and "builtin_metrics.clj" link points to specific commit, which would be better to point to master branch.</li>
</ul>


<p>    documentation/Acking-framework-implementation.html<br/>
    —</p>
<ul class="alternate" type="square">
	<li>"Storm's acker" and "execute method", and "put the record" link points to specific commit, which would be better to point to master branch.</li>
</ul>


<p>    contribute/Contributing-to-Storm.html<br/>
    —</p>
<ul class="alternate" type="square">
	<li>Modules built on top of Storm</li>
	<li>Do we still support @stormprocessor? If not, it would like to guide him/her to become part of storm-external.</li>
</ul>


<p>    Overall<br/>
    —</p>
<ul class="alternate" type="square">
	<li>All links which point to javadoc are broken. Can I expect it will work when site resources are deployed to official site?</li>
	<li>Minor: We would be better to create new page Kafka-and-Storm.html and replace Kestrel-and-Storm.html.</li>
	<li>Minor: How about removing all documents which are based on older than 0.9.0?</li>
</ul>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>Github user harshach commented on the pull request:</p>

<p>    <a href="https://github.com/apache/storm/pull/670#issuecomment-129465676" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/670#issuecomment-129465676</a></p>

<p>    @HeartSaVioR Thanks for the review. <br/>
    All links which point to javadoc are broken. Can I expect it will work when site resources are deployed to official site?<br/>
    Yes this will work once they deployed on the official site.<br/>
    Minor: We would be better to create new page Kafka-and-Storm.html and replace Kestrel-and-Storm.html.<br/>
    Since the patch is big and we are trying to use the existing content we probably should be file follow-up jiras  to replace them.<br/>
    Minor: How about removing all documents which are based on older than 0.9.0?<br/>
    I'll try to look into it but we should be doing a follow-up jiras to do this. Since this JIRA more about the redesign.</p>

<p>    Thanks.</p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>Github user harshach commented on the pull request:</p>

<p>    <a href="https://github.com/apache/storm/pull/670#issuecomment-129523372" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/670#issuecomment-129523372</a></p>

<p>    @HeartSaVioR Thanks. Will file a follow-up jira for the issues you raised.</p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>Github user harshach commented on the pull request:</p>

<p>    <a href="https://github.com/apache/storm/pull/670#issuecomment-129636717" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/670#issuecomment-129636717</a></p>

<p>    @ptgoetz can you take a look at it as well.</p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>Github user ptgoetz commented on the pull request:</p>

<p>    <a href="https://github.com/apache/storm/pull/670#issuecomment-130444965" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/670#issuecomment-130444965</a></p>

<p>    I have a few things that I think need to be fixed before we publish this. But rather than enumerate every last little thing and ask you to fix them, I'd rather jump in and fix them myself and let others do the same.</p>

<p>    I propose we move this off to an empty branch that just contains the storm website code &#8211; similar to the way github pages work. Then anyone could help fix things by opening a pull request against that branch. Then once we are all satisfied enough to publish it, we do so and remove the website code from the main source branches.</p>

<p>    I don't have a reference handy, but I think ASF INFRA has a way to publish web content from a specially-named branch (similar to how "gh-pages" branches work). That would eliminate the extra step of generating the site to SVN and committing that way. I'll look into this option some more.</p>

<p>    How does that sound?</p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>Github user harshach commented on the pull request:</p>

<p>    <a href="https://github.com/apache/storm/pull/670#issuecomment-130447771" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/670#issuecomment-130447771</a></p>

<p>    @ptgoetz if these issues are layout please list them out we can send a new patch. I don't think website layout needs to be blocked on docs or other fixes. I want to see this published soon rather than make it into feature branch and taking time to fix this. If its about docs we can file follow-up jiras to fix those. Even if its layout issues these should be minor and can be make it into follow-up jiras. It doesn't need to get into feature branch and IMHO its ready get published.</p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>Github user ptgoetz commented on the pull request:</p>

<p>    <a href="https://github.com/apache/storm/pull/670#issuecomment-130452561" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/670#issuecomment-130452561</a></p>

<p>    @harshach I'm not suggesting we create a feature branch, I'm suggesting we change the way we publish the site to make it easier for people to contribute to. I forgot to mention it before, but I'm happy to set that up.</p>

<p>    Some of the issues I see are minor layout issues (twitter feed should be below news, for example), some are maintenance-related (the recent news on the home page should be generated, not hard-coded in HTML), and some are related to ASF policy (we are pulling in content from facebook.com and google.com domains that potentially includes end-user tracking, and I'm not sure if that's allowed &#8211; I'd have to check).</p>

<p>    I'm not worried about docs/textual content, that can be fixed anytime.</p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>Github user harshach commented on the pull request:</p>

<p>    <a href="https://github.com/apache/storm/pull/670#issuecomment-130488992" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/670#issuecomment-130488992</a></p>

<p>    @ptgoetz will look into the fb & goog calls and also generating the news feed. As far as separate branch goes thats fine by me but another issue could be that the doc contributions will not be counted towards master branch rather into another branch and also merging those changes into another branch. </p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>Github user harshach commented on the pull request:</p>

<p>    <a href="https://github.com/apache/storm/pull/670#issuecomment-131827381" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/670#issuecomment-131827381</a></p>

<p>    @ptgoetz removed references to google & fb and news widget is on top. Can you take a look.</p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>Github user harshach commented on the pull request:</p>

<p>    <a href="https://github.com/apache/storm/pull/670#issuecomment-136895532" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/670#issuecomment-136895532</a></p>

<p>    @ptgoetz upmerged. Can you take a look.</p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>Github user ptgoetz commented on the pull request:</p>

<p>    <a href="https://github.com/apache/storm/pull/670#issuecomment-139618704" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/670#issuecomment-139618704</a></p>

<p>    +1</p>

<p>    I will merge this into a new empty "asf-site" branch and work with INFRA to setup git-based publishing, which will eliminate the extra svn step to publish the site.</p>

<p>    In the meantime, we can let contributors open up pull requests against the asf-site branch and start fixing some of the issues that still exist with the redesign.</p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>Github user ptgoetz commented on the pull request:</p>

<p>    <a href="https://github.com/apache/storm/pull/670#issuecomment-139621217" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/670#issuecomment-139621217</a></p>

<p>    @harshach This has been merged into the "asf-site" branch. Can you close this pull request?</p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>Github user harshach closed the pull request at:</p>

<p>    <a href="https://github.com/apache/storm/pull/670" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/670</a></p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>GitHub user ptgoetz opened a pull request:</p>

<p>    <a href="https://github.com/apache/storm/pull/764" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/764</a></p>

<p>    <a href="https://issues.apache.org/jira/browse/STORM-950" title="Apache Storm website redesign" class="issue-link" data-issue-key="STORM-950"><del>STORM-950</del></a>: Update website for publishing.</p>

<p>    <a href="https://issues.apache.org/jira/browse/STORM-950" class="external-link" rel="nofollow">https://issues.apache.org/jira/browse/STORM-950</a></p>

<p>    This pull request builds on the previous work:</p>

<ul>
	<li>adds a people section</li>
	<li>cleaned up/expanded talks and slideshows page</li>
	<li>Added documentation section for "external" components</li>
	<li>removed hard-coded links to news section</li>
	<li>Misc. minor fixes</li>
</ul>


<p>    Once this is in, we can publish via svn one last time, then ask INFRA to turn on git publishing so the site gets updated anytime the `asf-site` branch gets updated.</p>

<p>You can merge this pull request into a Git repository by running:</p>

<p>    $ git pull <a href="https://github.com/ptgoetz/storm" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/ptgoetz/storm</a> asf-site-ptgoetz</p>

<p>Alternatively you can review and apply these changes as the patch at:</p>

<p>    <a href="https://github.com/apache/storm/pull/764.patch" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/764.patch</a></p>

<p>To close this pull request, make a commit to your master/trunk branch<br/>
with (at least) the following in the commit message:</p>

<p>    This closes #764</p>

<hr />
<p>commit 565e354057f0f730313d53b75da8aefe97ffc632<br/>
Author: P. Taylor Goetz <ptgoetz@gmail.com><br/>
Date:   2015-09-17T16:48:29Z</p>

<p>    remove hard-coded news links</p>

<p>commit fde3433f9412c708fdefbdf78a7f638b6f831745<br/>
Author: P. Taylor Goetz <ptgoetz@gmail.com><br/>
Date:   2015-09-24T17:18:32Z</p>

<p>    add people section, clean up videos/slides section</p>

<p>commit af6d8ffc59fcc9d17a5fbdf911d6378aaccbe949<br/>
Author: P. Taylor Goetz <ptgoetz@gmail.com><br/>
Date:   2015-09-25T19:02:33Z</p>

<p>    add external module docs; misc tweaks</p>

<hr />

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>Github user harshach commented on the pull request:</p>

<p>    <a href="https://github.com/apache/storm/pull/764#issuecomment-143345191" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/764#issuecomment-143345191</a></p>

<p>    +1</p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>Github user Parth-Brahmbhatt commented on the pull request:</p>

<p>    <a href="https://github.com/apache/storm/pull/764#issuecomment-143346165" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/764#issuecomment-143346165</a></p>

<p>    +1.</p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>Github user d2r commented on the pull request:</p>

<p>    <a href="https://github.com/apache/storm/pull/764#issuecomment-143348789" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/764#issuecomment-143348789</a></p>

<p>    <a href="https://github.com/apache/storm/blob/1d09012e718091f67e3c8710fae7e6da64d86378/css/style.css#L341-L344" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/blob/1d09012e718091f67e3c8710fae7e6da64d86378/css/style.css#L341-L344</a></p>

<p>    Sorry, but could we tone down the button hover transition to 100 or 200ms from 1s?  It's pretty intense. <img class="emoticon" src="https://issues.apache.org/jira/images/icons/emoticons/smile.png" height="16" width="16" align="absmiddle" alt="" border="0"/></p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>Github user d2r commented on the pull request:</p>

<p>    <a href="https://github.com/apache/storm/pull/764#issuecomment-143353049" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/764#issuecomment-143353049</a></p>

<p>    Some of the links referencing code in Message-passing-implementation, storm-jdbc, Transactional-topologies look like they might be broken or pointing to the wrong place.  We can correct this separately.</p>

<p>    +1 even with these issues.  I think this looks really good, nice work.</p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>Github user asfgit closed the pull request at:</p>

<p>    <a href="https://github.com/apache/storm/pull/764" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/764</a></p>
